app/pb_kits/playbook/pb_date_picker/_date_picker.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= content_tag(:div,
+    aria: object.aria,
+    class: object.classname + object.error_class,
+    data: object.data,
+    id: object.id) do %>
+    <% if !object.hide_label %>
+      <%= pb_rails("caption", props: { text: object.label }) %>
+    <% end %>
+      <div class="input_wrapper">
+        <input
+            autocomplete="off"
+            id="<%= object.picker_id %>"
+            name="<%= object.name %>"
+            <%= object.required ? required="required" : ''%>
+        />
```

Comment:
Many of our form kits allow the user to bring their own input as they might need additional things added to the input that we don't enable through props. Do you think we would need that for this kit?

Also, do you think we can use the text input kit here instead?

Reply:
How is dark passed to the input?

Reply:
Removed html input tag and used text input kit instead.  Dark is now passed as a prop to the text input kit.